test/sagemaker_tests/huggingface_pytorch/training/integration/sagemaker/test_smdp.py

```diff
@@ -119,26 +95,29 @@ def test_smdp_question_answering(ecr_image, instance_type, py_version, sagemaker
 @pytest.mark.processor("gpu")
 @pytest.mark.skip_cpu
 @pytest.mark.skip_py2_containers
-def test_smdp_question_answering_multinode(ecr_image, instance_type, py_version, sagemaker_session, tmpdir):
+def test_smdp_question_answering_multinode(ecr_image, sagemaker_regions, py_version, ):
     """
     Tests SM Distributed DataParallel single-node via script mode
     """
+    invoke_hf_pt_helper_function(ecr_image, sagemaker_regions, _test_smdp_question_answering_function,
+                                 py_version, 2)
```

Comment:
Same here, function args in the form of dict should be used instead of passing separate function arguments.